include/libopencm3/stm32/common/dma2d_common_f47.h

```diff
@@ -145,7 +145,8 @@
 #define DMA2D_OPFCCR_CM_ARGB8888	0
 #define DMA2D_OPFCCR_CM_RGB888		1
 #define DMA2D_OPFCCR_CM_RGB565		2
-#define DMA2D_OPFCCR_CM_ARGB4444	3
+#define DMA2D_OPFCCR_CM_ARGB1555	3
+#define DMA2D_OPFCCR_CM_ARGB4444	4
```

Comment:
this one is perfect, I can pick that right now on the original f4 file if you like?

Reply:
i don't need it in the f4 file, but go ahead